www/artists/thall/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="stylesheet.css">
+
+</head>
+<body>
+  <li class="home">  <a href= "index.html"> Home</a> </li>
```

Comment:
You aren't allowed to have list items outside of either an unordered list or an ordered list. These list items are outside of an ordered or unordered list. Do you see the problem?